_posts/2019-04-16-sicko-mode.md

```diff
@@ -0,0 +1,100 @@
+Aubrey “Drake” Graham (hereby solely referred to as Drake), plays a close
+friend of our protagonist. He notes that as for heterosexuality, “that shit all
+he know, he don’t know nothin’ else”. Unable to see our hero living a lie,
+our artists take action by awakening the protaganist's true sexuality (“I
+tried to show him, yea I tried to show him”). Their attempt is successful, as
+they note our character is “goin’ on you with the pick and roll”. For readers
+unfamiliar with basketball, a pick and roll involves a player forcing a
+defender to *switch* to defending him by setting up a screen. These words are
+accompanied by a beat switch -- and also accompanied by our character’s switch
+of sexuality.
+
+The second beat is more aggressive and powerful, consistent with our
+character’s newfound interest in masculinity.
+
+> Different coloured chains, think my jeweller really sellin’ fruits
+
+ He immediately affirms his new sexuality by buying multicoloured jewellery,
+which is associated with gay pride. Furthermore “fruit” is common slang for
+homosexuals, further alluding to his jeweller selling symbols of homosexuality.
+
+> Most of these girls ain’t got a clue
+
+Not only does our character no longer see women as potential sexual partners,
+he goes so far as to deride their intelligence as a whole.
+
+> We ain’t even make it to room.  
+> She thought it was the ocean, it’s just a pool.
+
+Here, as a woman attempts to engage in promiscuous behaviour, our hero denies
+her access to his room (and therefore himself). Furthermore, he corrects her --
+his desire for women cannot be compared to the vast ocean, but just a small
+pool, by comparison practically non-existent.
+
+However, due to the anti-homosexual sentiments still present in many American
+communities, our protagonist is unable to sustain his true desires. Once they
+find out, he is physically assaulted and knocked “out like a light **(ey-uh)**”
+(emphasis mine). The beat change here again represents another change in his
+life. He must settle for practising his sexuality in secret and pretending to
+obey heterosexual norms.
+
+He regrets this as he notes “this shit got double bedroom”, meaning he
+unfortunately has to physically separate each of his sexualities - one bedroom
+for the women in his life, and another for the men. He must publicly proclaim
+“checks over stripes” (stripes on a multi coloured flag represent
+homosexuality). Despite his suffering, our strong and resilient protagonist
+still proclaims:
+
+> See the shots that I took (Ayy), wet like I'm Book (Ayy)  
+> Wet like I'm Lizzie
+
+or that he is able to get wet (i.e. ejaculate) both with Book (presumably a
+man) and Lizzie (presumably a woman). He is able to lead his double life well.
+
+Unfortunately, as the song title suggests, he is eventually unable to
+conclusively assert his preferences. He remains in fear of being branded a
+“sicko”, and this song is a sad reminder of that time his peers had thought he
```

Comment:
swap comma and quote

Reply:
https://english.stackexchange.com/questions/114375/how-to-use-quotes-in-a-list-with-commas

It is valid otherwise. Furthermore, when quoting in order to distinguish the word, I believe it does not make sense to have the comma inside. We are trying to stress the word "sicko" not the word "sicko,".

Reply:
That is a common misconception. Please see my attached link. You are quoting an excerpt, and as such your citation does not apply